### PR TITLE
Chore/cleanup

### LINKED ISSUE #49: Request for a Clean Demo Renderer
Demo Renderer is a good reference for institutes to start our own development. Appreciate it!
After we compiled the demo renderer and run it locally, there are many vulnerabilities and critical warnings found in the project.  Wonder whether GovTech can go extra miles to provide a clean copy of demo renderer for us to add-on?  Thanks.
image

### PR DESCRIPTION
**Pull request by yehjxraymond**
- npm audit fix
- integration test fix
- test fix
- removal of .DS_STORE

Fixes #49

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,10 @@ coverage/
 *.ipa
 *.apk
 
+# Rubbish
```
Comment (rjchow) [inline .gitignore]: a lot of the shit in the .gitignore is useless in our context, it was just copied and pasted off somewhere i think
probably want to clean it up

File: scripts/integration-headless.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+CI=false npm run build && \
```
Comment (rjchow) [inline scripts/integration-headless.sh]: what's CI=false?

Comment (yehjxraymond) [reply scripts/integration-headless.sh]: Travis treats warning message as error. So specifically for the build phase it allows warning message to pass. 

This is needed for some warning during the build

File: .gitignore (hunk 1 above)
Comment (yehjxraymond) [reply .gitignore]: Didnt notice. Fixed.

File: scripts/integration-headless.sh (hunk 2 above)
Comment (yehjxraymond) [reply scripts/integration-headless.sh]: https://github.com/facebook/create-react-app/issues/3657

**Pull request merged**